packit_service/worker/whitelist.py

```diff
@@ -101,12 +101,11 @@ def get_account(self, account_name: str) -> Optional[dict]:
 
     def add_account(self, github_app: InstallationEvent) -> bool:
         """
-        Add account to whitelist, if automatic verification of user
-        (check if user is packager in fedora) fails, account is still inserted in whitelist
-         with status : `waiting`.
-         Then a scripts in files/scripts have to be executed for manual approval
+        Add account to whitelist.
+        Status is set to approved_automatically' or 'waiting' (in case the account is not in FAS).
+
         :param github_app: github app installation info
-        :return: was the account (auto/already)-whitelisted?
+        :return: has the account been (auto/already)-whitelisted?
```

Comment:
:nerd_face: 

Okay, I looked up what is the difference between "was" and "has been". But b/c this is a method returning a Boolean flag indicating whether the action (whitelisting) happened or not (i.e. whitelisting was finished), I think "was" would be more correct.